src/patternfly/components/Form/form.scss

```diff
@@ -251,3 +295,162 @@
+    }
+  }
+}
+
+.pf-c-form__field-group-toggle {
+  grid-column: 1 / 2;
+  grid-row: 1 / 2;
+  padding-top: var(--pf-c-form__field-group-toggle--PaddingTop);
+  padding-right: var(--pf-c-form__field-group-toggle--PaddingRight);
+
+  &::after {
+    border-top: var(--pf-c-form__field-group-toggle--after--BorderTopWidth) solid var(--pf-c-form__field-group-toggle--after--BorderTopColor);
+    border-bottom: var(--pf-c-form__field-group-toggle--after--BorderBottomWidth) solid var(--pf-c-form__field-group-toggle--after--BorderBottomColor);
+  }
+
+  + .pf-c-form__field-group-header {
+    --pf-c-form__field-group-header--GridColumn: var(--pf-c-form__field-group-toggle--field-group-header--GridColumn);
+  }
+}
+
+.pf-c-form__field-group-toggle-button {
+  margin-top: var(--pf-c-form__field-group-toggle-button--MarginTop);
+  margin-bottom: var(--pf-c-form__field-group-toggle-button--MarginBottom);
+}
+
+.pf-c-form__field-group-toggle-icon {
+  display: inline-block;
+  min-width: var(--pf-c-form__field-group-toggle-icon--MinWidth);
+  text-align: center;
+  transition: var(--pf-c-form__field-group-toggle-icon--Transition);
+  transform: rotate(var(--pf-c-form__field-group-toggle-icon--Rotate));
+}
+
+.pf-c-form__field-group-toggle,
+.pf-c-form__field-group-header {
+  position: relative;
+
+  &::after {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    pointer-events: none;
+    content: "";
+  }
+}
+
+.pf-c-form__field-group-header {
+  grid-column: var(--pf-c-form__field-group-header--GridColumn);
+  grid-row: 1 / 2;
+  display: flex;
+  align-items: flex-start;
+  padding-top: var(--pf-c-form__field-group-header--PaddingTop);
+  padding-bottom: var(--pf-c-form__field-group-header--PaddingBottom);
+
+  &::after {
+    border-top: var(--pf-c-form__field-group-header--after--BorderTopWidth) solid var(--pf-c-form__field-group-header--after--BorderTopColor);
+    border-bottom: var(--pf-c-form__field-group-header--after--BorderBottomWidth) solid var(--pf-c-form__field-group-header--after--BorderBottomColor);
+  }
+}
+
+.pf-c-form__field-group-header-main {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+
+.pf-c-form__field-group-header-title {
+  display: flex;
+}
+
+.pf-c-form__field-group-header-title-text {
+  flex-grow: 1;
+}
+
+.pf-c-form__field-group-header-description {
+  margin-top: var(--pf-c-form__field-group-header-description--MarginTop);
+  color: var(--pf-c-form__field-group-header-description--Color);
+}
+
+.pf-c-form__field-group-header-actions {
+  margin-top: var(--pf-c-form__field-group-header-actions--MarginTop);
+  margin-bottom: var(--pf-c-form__field-group-header-actions--MarginBottom);
+  margin-left: var(--pf-c-form__field-group-header-actions--MarginLeft);
+  white-space: nowrap;
+}
+
+.pf-c-form__field-group-body {
+  grid-column: var(--pf-c-form__field-group-body--GridColumn);
+  grid-row: 2 / 3;
```

Comment:
```suggestion
```
I'd remove this, as it would allow multiple bodies and their auto-placement would be 2, 3, 4, etc